usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -202,10 +212,11 @@ def handle(self, *args, **options):
                     piid=row.get('piid'),
                     fain=row.get('fain'),
                     uri=row.get('uri'),
-                    parent_award_id=row.get('parent_award_id'))
+                    parent_award_id=row.get('parent_award_id'),
+                    use_cache=True)
                 award.latest_submission = submission_attributes
-                award.save()
-            except:
+                award_queue[award.manual_hash()] = award
+            except:   # TODO: silently swallowing a bare exception is bad mojo
```

Comment:
I think this is supposed to catch the exception being thrown above (if treasury_account is None) - can probably refactor that into an `ObjectDoesNotExist` exception. (Or just check if it is None in a finally block and in that case log the exception and move on)